basics_of_statistics_in_sst-core/statistics.md

```diff
@@ -0,0 +1,670 @@
+
+```
+ComponentName, StatisticName, StatisticSubId, StatisticType, SimTime, Rank, Sum.u64, SumSQ.u64, Count.u64, Min.u64, Max.u64, BinsMinValue.u64, BinsMaxValue.u64, BinWidth.u64, TotalNumBins.u32, NumActiveBins.u32, NumItemsCollected.u64, NumItemsBinned.u64, NumOutOfBounds-MinValue.u64, NumOutOfBounds-MaxValue.u64, Bin0:0-5.u64, Bin1:5-10.u64
+stats_example, , , Accumulator, 1000, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 1000, 0, 9, 81, 0, 0, 0, 0, 9, 5, 2, 1, 1, 1, 0, 0, 0, 1
+stats_example, , , Accumulator, 2000, 0, 1, 1, 2, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 2000, 0, 17, 145, 0, 0, 0, 0, 9, 5, 2, 1, 2, 2, 0, 0, 0, 2
+stats_example, , , Accumulator, 3000, 0, 3, 5, 3, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 3000, 0, 24, 194, 0, 0, 0, 0, 9, 5, 2, 1, 3, 3, 0, 0, 0, 3
+stats_example, , , Accumulator, 4000, 0, 6, 14, 4, 0, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 4000, 0, 30, 230, 0, 0, 0, 0, 9, 5, 2, 1, 4, 4, 0, 0, 0, 4
+stats_example, , , Accumulator, 5000, 0, 10, 30, 5, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 5000, 0, 35, 255, 0, 0, 0, 0, 9, 5, 2, 1, 5, 5, 0, 0, 0, 5
+stats_example, , , Accumulator, 6000, 0, 15, 55, 6, 0, 5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 6000, 0, 39, 271, 0, 0, 0, 0, 9, 5, 2, 2, 6, 6, 0, 0, 1, 5
+stats_example, , , Accumulator, 7000, 0, 21, 91, 7, 0, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 7000, 0, 42, 280, 0, 0, 0, 0, 9, 5, 2, 2, 7, 7, 0, 0, 2, 5
+stats_example, , , Accumulator, 8000, 0, 28, 140, 8, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 8000, 0, 44, 284, 0, 0, 0, 0, 9, 5, 2, 2, 8, 8, 0, 0, 3, 5
+stats_example, , , Accumulator, 9000, 0, 36, 204, 9, 0, 8, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 9000, 0, 45, 285, 0, 0, 0, 0, 9, 5, 2, 2, 9, 9, 0, 0, 4, 5
+stats_example, , , Accumulator, 10000, 0, 45, 285, 10, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 10000, 0, 45, 285, 0, 0, 0, 0, 9, 5, 2, 2, 10, 10, 0, 0, 5, 5
+stats_example, , , Accumulator, 11000, 0, 45, 285, 10, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 11000, 0, 45, 285, 0, 0, 0, 0, 9, 5, 2, 2, 10, 10, 0, 0, 5, 5
+stats_example, , , Accumulator, 11000, 0, 45, 285, 10, 0, 9, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+stats_example, , , Histogram, 11000, 0, 45, 285, 0, 0, 0, 0, 9, 5, 2, 2, 10, 10, 0, 0, 5, 5
+```
+## Final Notes
+
+As of SST V10 there are a few idiosyncrasies to keep in mind when working with statistics in SST.  The noted items may explain unusual behavior seen when using statistics but will not prevent using them.
```

Comment:
These may be bugs in SST. We are tracking these down. In general, it'd be helpful to open a ticket (issue) on sst-core if you encounter unexpected behavior like this.